server/db/index.js

```diff
@@ -0,0 +1,28 @@
+//collect all models and db connection
+//export everything in the database from here
+const db = require('./database');
+const { User, Order, OrderDetail, Product, Wishlist } = require('./models/index');
+
+//set up associations here
+//Wishlist gets the association key as userId
+User.hasMany(Wishlist);
+//Wishlist gets the association key as productId
+Product.hasMany(Wishlist);
+//Orders gets the association key as userId
+User.hasMany(Order);
+//OrderDetails gets the association key as orderId
+Order.hasMany(OrderDetail);
+//OrderDetails gets the association key as productId
+Product.hasMany(OrderDetail);
```

Comment:
You seem to be missing the other side of these relations... You should always set both the `source` and the `target` relations. Here for example:

```javascript
User.hasMany(Wishlist);
Wishlist.belongsTo(User);
```

Reply:
@wulalasuperdope . I'll make this change when i make the PR for the seed branch.